_docs/_latest/configure/scanner/rules.md

```diff
@@ -251,7 +253,7 @@ To enable the blacklist notifier, add the followings to the notifier section in
     - member_email: '@gmail.com'
 ```
 
-## GCS bucket ACL rules
+## Cloud Storage bucket ACL rules
```

Comment:
We'll need to make sure this doesn't break any direct links, e.g. search for 

`#gcs-bucket-acl-rules`

Reply:
Agreed we'll need to change this one since we shouldn't use an abbreviation for the product name even in the link. Can we do a redirect on the anchor (which may be a simpler solution for the previous anchor, too)?

Reply:
Resolved.